tests/base.py

```diff
@@ -32,10 +32,10 @@ def tile_values(fd):
 
         def adgroup_values(fd):
             for line in fd:
-                row = [el.decode('utf-8') for el in line.split(',')]
+                locale, check_inadjacency_str = [el.decode('utf-8') for el in line.split(',')]
                 yield dict(zip(
-                    ('locale',),
-                    row))
+                    ('locale', 'check_inadjacency'),
+                    (locale, True if check_inadjacency_str == 'true' else False)))
```

Comment:
nit:  
True if check_inadjacency_str == 'true' else False
is the same as
check_inadjacency_str == 'true'